.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
I wouldn't ask the user to do that because the user might not know for sure (i.e. it's not necessarily easy for the user to decide whether something is supposed to work or not.)

Most templates have something along the lines of: what were you trying to do? What did you expect to happen? What actually happened.